### PR TITLE
[GSoC] Updated Docs

### PR DESCRIPTION
**Pull request by madhurgupta10**
Updated Docs

### CODE REVIEW

File: DB_Schema/leaderboard.md [hunk 1]
```diff
@@ -55,5 +114,47 @@ Note: make sure to use `__` double underscore
 ## To Create Tables Run a SQL file like
 
 ```
-mysql -u root -p --no-data dbname > create_leaderboard_tables.sql
+$ mysql -u root -p --no-data dbname > create_leaderboard_tables.sql
```
Comment (nicolas-raoul) [inline DB_Schema/leaderboard.md:117]: I think this command would overwrite the SQL file rather than read from  it.

Comment (madhurgupta10) [reply DB_Schema/leaderboard.md:117]: This one is actually if someone wants to create a new database and tables in their own toolsforge

Comment (madhurgupta10) [reply DB_Schema/leaderboard.md:117]: @nicolas-raoul mybad I did not notice the arrow

File: DB_Schema/leaderboard.md [hunk 2]
```diff
@@ -55,5 +114,47 @@ Note: make sure to use `__` double underscore
 ## To Create Tables Run a SQL file like
 
 ```
-mysql -u root -p --no-data dbname > create_leaderboard_tables.sql
+$ mysql -u root -p --no-data dbname < create_leaderboard_tables.sql
```
Comment (nicolas-raoul) [inline DB_Schema/leaderboard.md:117]: Thanks!
Also, is `create_leaderboard_tables.sql` in Git? Please link to it here.

Comment (madhurgupta10) [reply DB_Schema/leaderboard.md:118]: I added the cd command to be clear

Comment (nicolas-raoul) [reply DB_Schema/leaderboard.md:118]: Great, thanks :-)

**Comment by nicolas-raoul** [comment]
Thanks!

**Pull request merged**